app/src/main/java/org/breezyweather/remoteviews/config/AbstractWidgetConfigActivity.kt

```diff
@@ -189,11 +188,6 @@ abstract class AbstractWidgetConfigActivity : GeoActivity() {
             initView()
             updateHostView()
         }
-    }
-
-    override fun onPostCreate(savedInstanceState: Bundle?) {
-        super.onPostCreate(savedInstanceState)
-
         KeyboardResizeBugWorkaround.assistActivity(this)
```

Comment:
Moving this into `onCreate` fixes an issue where the window insets are only correctly applied after tapping into the text input field for the custom subtitle ([issue](https://github.com/user-attachments/assets/2284d638-a65e-4c54-a172-6901c02bbdd2), [fix](https://github.com/user-attachments/assets/1ea4e40c-c992-4957-946d-9ef43e6f88b1)).